beacon-chain/execution/payload_body_test.go

```diff
@@ -0,0 +1 @@
+package execution
```

Comment:
```suggestion
package execution_test
```

Prefer blackbox test, if possible

Reply:
Why blackbox? All the types in payload_body.go are private.

Reply:
OK that is good enough reason. I have seen a lot of tests for public APIs only in prysm that were not using blackbox so I :scream_cat: